packages/rdbg/rdbg.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "erwan.jahier@imag.fr"
+bug-reports: "erwan.jahier@imag.fr"
+authors: [ "Erwan Jahier" ]
+license: "GPL-3"
+homepage: "http://rdbg.forge.imag.fr/"
+dev-repo: "https://forge.imag.fr/anonscm/git/rdbg/rdbg.git"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "rdbg-plugin"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-unix"
+  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
```

Comment:
On Thu, 13 Aug 2015, David Sheets wrote:

> Previously, your package was determined to be uninstallable due to the camlp4
> constraints.
> 
> Now, your package is installable but contains issues later in the install process. In
> particular, your archive contains more than 1 top level directory. Your archive should
> contain only a single top level directory.

ok but previously, 6/8 builds passed, and now that I have fixed an 
issue, only 2/8 passe, which is confusing.

Anyway, I've fixed the multiple directory issue, and now its looks 
better (except for the mac build).

Thanks.

Just a last question about the publishing process. Just tell if I'm 
wrong :

1: opam publish
2: if not all build tests are green (except the 8th), fix the pbs 
without waiting for feedback and goto 1
3: wait for someone with superpowers to the merge the request

-- 
R1.